src/scores/probability/brier_impl.py

```diff
@@ -65,3 +72,160 @@ def brier_score(
         check_binary(obs, "obs")
 
     return mse(fcst, obs, reduce_dims=reduce_dims, preserve_dims=preserve_dims, weights=weights)
+
+
+def brier_score_for_ensemble(
+    fcst: XarrayLike,  # type: ignore
+    obs: XarrayLike,  # type: ignore
+    ensemble_member_dim: str,
+    event_thresholds: Union[float, int, Sequence[float]],
+    *,  # Force keywords arguments to be keyword-only
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[xr.DataArray] = None,
+    fair_correction: bool = True,
+    event_threshold_mode: Callable = operator.ge,
```

Comment:
Should this be `event_threshold_operator` rather than `event_threshold_mode`? No strong opinion here, just a thought. Not sure what we might have done elsewhere.

Reply:
ping @nicholasloveday  I think this may have gotten lost

Reply:
I think this is a nice suggestion and I have updated it